contents/docs/advanced/proxy/vercel.mdx

```diff
@@ -17,7 +17,7 @@ Vercel supports [rewrites](https://vercel.com/docs/concepts/projects/project-con
 }
 ```
 
-Some frameworks, like **SvelteKit** and **Astro**, require a hungrier regex pattern like:
+Some frameworks, like **SvelteKit** and [Astro](/tutorials/astro-analytics), require a hungrier regex pattern like:
```

Comment:
What does hungrier mean here lol. Can we update it?

Reply:
It just means it matches more things, was a recommended term by people who suggested this change.